### DISCUSSION TITLE
Testing multiple domains in local dev env.

### DISCUSSION THREAD

**Discussion by sunglee** [vercel/next.js#15376] [Help]
Hi,

Is it possible to use multiple domain names pointing to localhost in dev mode?
For example, while developing, both foo.com and bar.com point to http://localhost:300. Is it doable?

Thanks!

**Comment by ArenoRobert** [comment]
Hi @sunglee !
Did you find a solution for this?

Best
Robert

**Comment by kuangthien** [comment]
I also wanna test local env with a domain which I config in etc/hosts
Does anyboby has solution?

**Comment by Bfaschat** [comment]
Also wondering if this is possible
